Dockerfile

```diff
@@ -3,14 +3,14 @@ MAINTAINER Dimagi <devops@dimagi.com>
 
 ENV PYTHONUNBUFFERED=1 \
     PYTHONUSERBASE=/vendor \
-    PATH=/vendor/bin:$PATH
+    PATH=/vendor/bin:$PATH \
+    NODE_VERSION=5.12.0
 
-RUN curl -sL https://deb.nodesource.com/setup_5.x | bash - \
- && apt-get -y install \
-    nodejs \
- && apt-get clean \
- && rm -rf /var/lib/apt/lists/* \
- && mkdir /vendor
+RUN mkdir /vendor
+
+RUN curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-x64.tar.gz" \
```

Comment:
Why separate this into two separate `RUN` commands? It's my understanding that each command creates a new layer, and it's desirable to reduce the number of layers.